### ISSUE TITLE
Remove EvilGinx2 IOC Header

### ISSUE THREAD

**Issue by s1lvershad0w** [fin3ss3g0d/evilgophish#47]
## Current Behavior

- Affected file: [http_proxy.go](https://github.com/kgretzky/evilginx2/blob/master/core/http_proxy.go)
- Header value that could be a possible IOC: **e := []byte{208, 165, 205, 254, 225, 228, 239, 225, 230, 240}**
- This value decodes to : **X-Evilginx**

## Possible Solution

Remove setting this header : `req.Header.Set(string(e), e_host)` by deleting lines 613 to 617

## Steps to Reproduce

1. `grep -in '0x88' .evilgophish/evilginx2/core/http_proxy.go`
2. Copy the header value from the above line no.

![IOC](https://user-images.githubusercontent.com/84075091/224933584-17b41d59-f885-46f4-b16a-94a7c649c4f7.png)

3. Visit [play.golang.org](https://go.dev/play/)
4. Decode the header value with the below Go function:
```
package main

import "fmt"

func main() {
	hg := []byte{208, 165, 205, 254, 225, 228, 239, 225, 230, 240}

	for n, b := range hg {
		hg[n] = b ^ 0x88
	}

	fmt.Println(string(hg))
}
```

5. Click "Run"

**Comment by fin3ss3g0d** [comment]
Good find. Original author is an asshole for leaving this in there.

**Comment by s1lvershad0w** [comment]
Thanks, and yes he is haha.

Got burnt bad on my last Op, did some digging and found this. Hopefully it
helps others.

**Comment by fin3ss3g0d** [comment]
Appreciate you pointing that out, it has been fixed in 186e8cfef5871c73d3dd553b9edb39ae24d06021
